# 1-takeoff  2-landing
int8 takeoffOrLanding  

---

#1-success
int8  ack